debian/update.sh

```diff
@@ -60,15 +60,20 @@ if [ -n "$PIKERNELMODDIR" ]; then
     fi
 fi
 
+HOST_ARCH="$(uname -m)"
 ARCH=${ARCH:-arm}
 case "$ARCH" in
 arm)
-    CROSS_COMPILE=${CROSS_COMPILE:-arm-linux-gnueabihf-}
+    if ! grep -q -E "armv(6l|7|7l)" <<<"$HOST_ARCH"; then
```

Comment:
armv6l?

Reply:
Yes, made me also wonder, but @naruxde validated this for me on a Core